Source/SessionManager.swift

```diff
@@ -606,28 +633,22 @@ open class SessionManager {
         var HTTPBodyStream: InputStream?
```

Comment:
Great catch! I actually deleted the variable altogether in one of the downstream PRs. I'll just leave it as is for now if that's cool.